app/controllers/idv/gpo_confirm_cancel_controller.rb

```diff
@@ -0,0 +1,15 @@
+module Idv
+  class GpoConfirmCancelController < ApplicationController
+    include IdvSession
+    include StepIndicatorConcern
+    include GoBackHelper
+
+    before_action :confirm_two_factor_authenticated
+    before_action :confirm_idv_needed
+
+    def index
+      @go_back_path = idv_session.go_back_path || idv_path
```

Comment:
I looked into `idv_session.go_back_path` and it looks like it's set in CancellationsController. Is this code path going through there somehow?

Reply:
It's used in the `_cancel` shared view, but you're right we might be able to do without this at all. I'll look into it.